cmd/bootstrapper/generator.go

```diff
@@ -179,6 +179,11 @@ func queryBitcoin(ctx context.Context, client *http.Client, targetUrl string) (*
 	if err != nil {
 		return nil, fmt.Errorf("http get latest bitcoin block: %w", err)
 	}
+	if resp.StatusCode != 200 {
+		return nil, fmt.Errorf(
+			"http get latest bitcon block finished without success for %q with status code %q",
+			targetUrl, resp.Status)
```

Comment:
NIT: shorter error messages are easier to wrap by callers and extend with info 🙂 

```suggestion
		return nil, fmt.Errorf("http get latest bitcon block status error: %q", resp.Status)
```